src/imitation/scripts/config/train_adversarial.py

```diff
@@ -47,6 +47,7 @@ def train_defaults():
   checkpoint_interval = 5  # num epochs between checkpoints (<=0 disables)
   init_tensorboard = False  # If True, then write Tensorboard logs.
   rollout_hint = None  # Used to generate default rollout_path
+  working_dir_hint = ""  # Automatically set to working dir by `parallel.py`
```

Comment:
I'm a bit confused by the notion of a working directory here. I can imagine two possible designs, both reasonable:
  1. Rollouts can be in arbitrary locations, but you assume a particular directory structure within that. In this case, I'd favour having a config attribute e.g. `rollout_root`, which is currently assumed to be `{working_dir_hint}/data/expert_models`.
  2. All outputs are in a single directory. In this case, let's specify this root, which is currently assumed to be `{working_dir_hint}/data`. We should also then change it so that `log_root` is some subdirectory of this.

Reply:
I was imagining design (1). 

Replaced `working_dir_hint` a new config attribute `data_dir = "data/"`. `parallel.py` now manipulates this variable instead, and this is used to build the default rollout path.